www/Gemfile.lock

```diff
@@ -1,275 +0,0 @@
-PATH
```

Comment:
Hey @adamleff, I have reorganized and made the recommended changes to this PR. I belive I have reverted the Gemfile.lock, please let me know if it has not been reverted. 
Please review it when you get a chance. Thank you